stories/index.js

```diff
@@ -141,6 +141,9 @@ storiesOf('FlipMove', module)
 
     return <HandleEmpty />
   })
+  .add('maintain container height', () => (
+    <Controls maintainContainerHeight={true} style={{border: 'solid 2px magenta'}} />
```

Comment:
Can you add some padding to this story? I checked it locally and it does work :) just wanna make sure the story includes it.